poetry.lock

```diff
@@ -16,7 +16,7 @@ files = [
 dev = ["cloudpickle", "coverage[toml] (>=5.0.2)", "furo", "hypothesis", "mypy (>=0.900,!=0.940)", "pre-commit", "pympler", "pytest (>=4.3.0)", "pytest-mypy-plugins", "sphinx", "sphinx-notfound-page", "zope.interface"]
```

Comment:
The lockfile should not be changed in this PR.

Reply:
Good point. This is a residue from introducing hypothesis, which I did in this PR first, and then pulled into a separate PR once I realized that it would make sense to have available before we merge this here.